bigtable/docs/snippets.py

```diff
@@ -79,13 +81,24 @@ def setup_module():
     operation = Config.INSTANCE.create(clusters=[cluster])
     # We want to make sure the operation completes.
     operation.result(timeout=100)
+    INSTANCES_TO_DELETE.append(Config.INSTANCE)
 
 
 def teardown_module():
-    Config.INSTANCE.delete()
+    for instance in INSTANCES_TO_DELETE:
+        if instance.exists():
+            instance.delete()
+
+    # Avoid any leak of snippet instances.
+    client = Client(admin=True)
+    (instances_list, failed_locations_list) = client.list_instances()
+    for instance in instances_list:
+        if "snippet-tests" in instance.instance_id:
+            instance.delete()
```

Comment:
This part should not be in the script, as we aren't creating such an instance anywhere.

Reply:
Now snippet instance is renamed to snippet-tests line 41.
Just to ensure any previously executed snippets instances pending should be deleted.

Reply:
I saw that change, but we can't remove instances that weren't created as part of this CI run:  we often have multiple PRs running CI at the same time.  Please remove this block.